### ISSUE TITLE
severe bug in gaussint

### ISSUE THREAD

**Issue by GillesDuvert** [gnudatalanguage/gdl#1691]
GDL> X = [1.00, 1.05, 1.27, 1.34, 1.38, 1.50] 
GDL> plot,x,gaussint(x)
% GAUSSINT: Variable is undefined: <Undefined>

**Comment by GillesDuvert** [comment]
closed #1692

**Comment by alaingdl** [comment]
Several serious bugs were detected, related to this one : 

```
GDL> print, GAUSSINT(replicate(0.5,3))
     0.691462     0.478120     0.478120
GDL> print, BESELI(findgen(3))
      1.00000
GDL> print, BESELJ(findgen(3))
      1.00000
[...]
IDL> print, GAUSSINT(replicate(0.5,3))
     0.691462     0.691462     0.691462
IDL> print, BESELI(findgen(3))
       1.0000000       1.2660658       2.2795854
IDL> print, BESELJ(findgen(3))
       1.0000000      0.76519769      0.22389078
```

**Comment by alaingdl** [comment]
T_PDF too is wrong in dims (to be add to TEST_MATH_FUNCTION_DIM)

**Comment by GillesDuvert** [comment]
@alaingdl thanks for all these tests.

**Comment by alaingdl** [comment]
Several cases were solved in #1702 
New test cases for 1 or 2 inputs in `test_math_function_dim.pro`

Please open a new issue if similar problems
